libraries/log4j.rb

```diff
@@ -15,19 +15,19 @@ def render_appender(name, options)
         when :layout
           content += render_layout(prefix, value)
         else
-          if value.respond_to?(:call)
-            content << %(#{prefix}.#{camelcase(key)}=#{value.call})
-          else
-            content << %(#{prefix}.#{camelcase(key)}=#{value})
-          end
+          content << if value.respond_to?(:call)
+                       %(#{prefix}.#{camelcase(key)}=#{value.call})
+                     else
+                       %(#{prefix}.#{camelcase(key)}=#{value})
+                     end
```

Comment:
`Style/ConditionalAssignment` can force the original with `EnforcedStyle: assign_inside_condition` if you would like :)